aptos-move/move-examples/staking/sources/oracle.move

```diff
@@ -0,0 +1,50 @@
+module staking::oracle {
+    use aptos_std::math128;
+    use pyth::i64;
+    use pyth::price;
+    use pyth::price_identifier;
+    use pyth::pyth;
+
+    friend staking::commission;
+
+    const PRECISION: u128 = 100000000; // 1e8
+    const INITIAL_MAX_AGE_SECS: u64 = 120; // 2 minutes
+    const PYTH_APT_ID: vector<u8> = x"03ae4db29ed4ae33d323568895aa00337e658e348b37509f5372ae51f0af00d5";
+    const PYTH :vector<u8> = b"pyth";
+
+    /// Price read from oracle is stale
+    const ESTALE_PRICE: u64 = 1;
+
+    struct OracleConfig has key {
+        max_age_secs: u64,
+    }
```

Comment:
Do you want this to be an enum?  In the event, future `OracleConfig`s would be different?

Reply:
Not for now. This is also just an example